opentelemetry-sdk/src/metrics/reader.rs

```diff
@@ -144,6 +145,7 @@ impl AggregationSelector for DefaultAggregationSelector {
             | InstrumentKind::UpDownCounter
             | InstrumentKind::ObservableCounter
             | InstrumentKind::ObservableUpDownCounter => Aggregation::Sum,
+            InstrumentKind::Gauge => Aggregation::LastValue,
```

Comment:
Exporters need no change, as the aggregation is anyway LastValue, right?

Reply:
Yes, the default is LastValue, and since both Gauge and Observable Gauge are absolute values, the aggregation and exporters needn't change.